vsintegration/src/FSharp.Editor/CodeLens/FSharpCodeLensService.fs

```diff
@@ -61,7 +61,11 @@ type internal FSharpCodeLensService
 
             override __.VisitTypeAbbrev( _, range) = Some range
 
-            override __.VisitLetOrUse(binding, range) = Some range
+            override __.VisitLetOrUse(binding, range) = 
+                match binding |> Seq.tryFind (fun b -> b.RangeOfHeadPat.StartLine = pos.Line) with
+                | Some entry ->
+                    Some entry.RangeOfBindingAndRhs
+                | _ -> Some range // This can be invalid and should not happen
```

Comment:
"This can be invalid and should not happen" - What if it does happen? What will be the behavior?

Also can you replace `_` with `None`?

Reply:
OK